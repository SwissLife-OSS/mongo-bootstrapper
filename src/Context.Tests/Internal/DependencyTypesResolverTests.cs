using System.Collections.Generic;
using System.Linq;
using Snapshooter.Xunit;
using Xunit;

namespace MongoDB.Extensions.Context.Tests.Internal;

[Collection("Sequential")]
public class DependencyTypesResolverTests
{
    [Fact(Skip = "Flaky test")]
    public void GetAllowedTypesByDependencies_All_Successful()
    {
        // Arrange

        // Act
        IEnumerable<string> knownNamespaces = DependencyTypesResolver
            .GetAllowedTypesByDependencies(new[] { "Coverlet" })
            .OrderBy(x => x);

        // Assert
        Snapshot.Match(knownNamespaces);
    }
}
