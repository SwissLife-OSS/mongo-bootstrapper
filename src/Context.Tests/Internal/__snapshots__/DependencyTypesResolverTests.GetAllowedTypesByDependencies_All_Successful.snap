[
  "Autofac",
  "CommandLine",
  "CSharpx",
  "JetBrains",
  "MongoDB",
  "Proxy",
  "RailwaySharp",
  "Serilog",
  "Snapshooter",
  "Squadron",
  "Xunit"
]
